cache/disk/lru.go

```diff
@@ -4,6 +4,26 @@ import (
 	"container/list"
 	"errors"
 	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	gaugeCacheSizeBytes = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "bazel_remote_disk_cache_size_bytes",
+		Help: "The current number of bytes in the disk backend",
+	})
+
+	counterEvictedBytes = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "bazel_remote_disk_cache_evicted_bytes_total",
+		Help: "The total number of bytes evicted from disk backend, due to full cache",
+	})
+
+	counterOverwrittenBytes = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "bazel_remote_disk_cache_overwritten_bytes_total",
+		Help: "The total number of bytes removed from disk backend, due to put of already existing key",
+	})
```

Comment:
What are we aiming to count here- attempted duplicate Puts, or duplicate Puts that we didn't short-circuit?

Reply:
My users does not understand this value on the dashboard, but for me it is interesting to see how common it is for the bazel client to re-upload same data several times. Either in the same session by concurrently executed actions, or by later sessions not checking if file exists before uploading. I had this patch since long time ago, before short-circuit functionality in 2d9211c3. Perhaps it could be interesting with a counter for skipped ones, to get the whole picture? Or do you have other suggestion?

Reply:
I think it's ok not to count skipped uploads, they're a part of normal operation from the point of view of the server.